### DISCUSSION TITLE
How about add a nightly build feed?

### DISCUSSION THREAD

**Discussion by BiDuang** [ShapeCrawler/ShapeCrawler#596] [Ideas]
Just like `Avalonia Nightly`, is it possible to make a nightly build feed and allow users trying the experiment features?

**Comment by AdamShakhabov** [comment]
Sounds interesting🤔 @BiDuang , thank you for the idea. I think we can do something like that.

**Reply by AdamShakhabov** [reply]
The script `create-package.bat` has been added to enable package generation from the `master` branch. Please check [Develop NuGet package](https://github.com/ShapeCrawler/ShapeCrawler#develop-nuget-package) for more information.

It doesn't yet have full automation like the Avalonia project. I think we will implement this later. I tried to do this via GitHub Packages, but I had issue with the public sharing token even with only `read` permissions.

**Reply by BiDuang** [reply]
Oh, installing NuGet package for local is not convenient, so how about using `pre-release` built by specific master commit's build as `beta` build?

If we use a version name looks like `1.1.0-foobar-beta` or something, then NuGet will mark it as `Pre-Release` so we can download it from  main NuGet feed instead of set up a local feed. [Microsoft Pre-release version](https://learn.microsoft.com/en-us/nuget/create-packages/prerelease-packages)

**Reply by AdamShakhabov** [reply]
Done. Please check [Prerelease Version](https://github.com/ShapeCrawler/ShapeCrawler#prerelease-version) guide.

Thank you for the advice!
